### PR TITLE
NEPT-1083: Implement the new nav tabs component 

### PR DESCRIPTION
**Pull request by gillesdeudon**
### Description

Implement the new header search bars component

### Change log

    Added: Theme implementation for the menu local task with the ecl component.

### Commands

No specific command to execute

### CODE REVIEW

File: templates/menu_local_task/menu-local-task.tpl.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
```
Comment (imanoleguskiza) [inline templates/menu_local_task/menu-local-task.tpl.php]: This file is exactly the same as the base atomium tpl so it should be removed.

File: templates/menu_local_tasks/menu-local-tasks.tpl.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Contains template file.
+ */
+?>
+<?php if ($primary): ?>
+  <nav class="ecl-navigation-wrapper">
+    <h2 class="ecl-sr-only"><?php print t('Primary tabs') ?></h2>
```
Comment (imanoleguskiza) [inline templates/menu_local_tasks/menu-local-tasks.tpl.php]: I'm a bit worried about adding string values directly into the tpl. Although its not a blocker I would like to know the opinion of the rest on this

**Comment by imanoleguskiza** [comment]
👍

**Comment by netlooker** [comment]
👍

**Pull request merged**